docs/compatibility.md

```diff
@@ -3,7 +3,7 @@
 StableHLO is a backward compatible ML compute opset inspired by HLO/MHLO.
 This document explains the kind and the extent of the compatibility guarantees
 that StableHLO provides, based on the process established in
-[the compatibility RFC](../rfcs/20220912-compatibility.md).
+[the compatibility RFC](https://github.com/openxla/stablehlo/tree/main/rfcs/20220912-compatibility.md).
```

Comment:
Based on your description, it looks like this can also be relative path.

Reply:
It can't because this is under `stablehlo/rfcs` rather than `stablehlo/docs`. Only paths under `docs` can be relative. Which part of the description is inaccurate?

Reply:
Nvm. I thought `rfcs/` was nested under `docs/` :)

Reply:
Ah, makes sense. No worries. TBH I think it would make sense to nest `rfcs` under `docs`, maybe that can be a follow-up change.